usagov/usagov-2021
# USAGOV-1038-CEO-PA-addresses

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1038

## Description

Check for levels before using levels[0]

## How to Test
Use the Contact Elected Officials tool with a PA address and you should see results where the tool on production currently shows no results.

Here is a url with an example address:
http://localhost/elected-officials-results?input-street=123+fake+st&input-city=philadelphia&input-state=PA&input-zip=12345#skip-to-h1
https://www.usa.gov/elected-officials-results?input-street=123+fake+st&input-city=philadelphia&input-state=PA&input-zip=12345#skip-to-h1

## Checklist for the Developer



## Checklist for the Peer Reviewers
